api/src/main/java/org/pmiops/workbench/reporting/ReportingUploadService.java

```diff
@@ -9,4 +11,6 @@
  */
 public interface ReportingUploadService {
   void uploadSnapshot(ReportingSnapshot reportingSnapshot);
+
```

Comment:
Please put a javadoc that this `captureTimestamp` must match what's in the snapshot (until it's removed), and all records must come from a single transaction.